### PR TITLE
Replay & Metadata Url format

### PR DESCRIPTION
**Pull request by Philaeux**
Related to #9

### CODE REVIEW

**Comment by rossengeorgiev** [comment]
I don't like these at a instance methods. Maybe there needs to be a until subpackage to hold these.

**Comment by Philaeux** [comment]
What about a `helpers.py` module at the `dota2/dota2/` level, next to `enums.py` ?

**Comment by rossengeorgiev** [comment]
I prefer the way its done in the `steam` package. There is a `util` subpackage that contains various helper functions.

**Comment by rossengeorgiev** [comment]
Hi, are you planning to finalize this PR ?

**Comment by Philaeux** [comment]
Sorry forgot this was on the fire. I will move them as stated in your comment https://github.com/ValvePython/dota2/pull/18#issuecomment-277550059 

The part with the secret will still be open for a new PR.

File: dota2/util/__init__.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    """Format the url to download a match replay
+
+    :param match_id: match id
+    :type match_id: :class: `int`
+    :param cluster: cluster the match is saved on
+    :type cluster: :class: `int`
+    :param replay_salt: salt linked to the replay
+    :type replay_salt: :class: `int`
+    :return url to download the replay of a specific match
+    :rtype :class: `str`
```
Comment (rossengeorgiev) [inline dota2/util/__init__.py]: Function also retrusn `None`, which is not mentioned

Comment (Philaeux) [inline dota2/util/__init__.py]: When does str.format return None? Should I test the arguments ?

**Comment by rossengeorgiev** [comment]
Merged in 598717e

**Pull request closed without merging**